Mynewt 1.8.0

szymon-czapracki opened the pull request.

szymon-czapracki committed:
[dpl] Don't redefine panic assert from mynewt-core

szymon-czapracki committed:
Change config to update to mynewt-core 1.8.0

szymon-czapracki commented:
@ncasaril FYI

ncasaril commented:
Would those packages that today use newtmgr have to change to use smp? For instance: 

- sys/uwbcfg
- mgmt/bcast_ota
- lib/nmgr_uwb
- lib/nmgr_cmds

szymon-czapracki committed:
[lib] Convert nmgr_uwb to smp_uwb

szymon-czapracki committed:
[lib] Convert nmgr_cmds to smp_cmds

szymon-czapracki committed:
[apps/syscfg] Update syscfg

Update package names in pkg.yml for smp ones.
Update syscfg.yml with vals from 1.8.0 core

szymon-czapracki committed:
[apps/twr_aloha] Update twr_aloha

szymon-czapracki committed:
[apps/twr_tag_tdma] Update twr_tag_tdma

szymon-czapracki committed:
[sys/uwb_cfg] Use smp instead of nmgr

szymon-czapracki committed:
[mgmt/bcast_ota] Use smp instead of nmgr

szymon-czapracki committed:
[hw/bsp] Use smp instead of nmgr

szymon-czapracki committed:
[porting/dpl] Use smp instead of nmgr

szymon-czapracki committed:
[hw/drivers] Use smp instead of nmgr

szymon-czapracki committed:
[lib/panmaster] Use smp instead of nmgr

szymon-czapracki committed:
[lib/uwb_transport] Use smp instead of nmgr

ncasaril commented on apps/twr_tag_tdma/pkg.yml:
```diff
@@ -32,9 +32,11 @@ pkg.deps:
+#   - "@apache-mynewt-core/encoding/json"
 #   - "@apache-mynewt-core/mgmt/imgmgr"
-    - "@apache-mynewt-core/mgmt/newtmgr"
+#   - "@apache-mynewt-core/mgmt/newtmgr"
```
Don't leave the old newtmgr here as a comment, just remove it.

szymon-czapracki replied:
Done

ncasaril commented on apps/syscfg/pkg.yml:
```diff
@@ -35,7 +35,9 @@ pkg.deps:
     - "@apache-mynewt-core/sys/stats/full"
 #   - "@apache-mynewt-core/mgmt/imgmgr"
-    - "@apache-mynewt-core/mgmt/newtmgr"
+#   - "@apache-mynewt-core/mgmt/newtmgr"
```
Don't leave the old newtmgr here as a comment, just remove it.

szymon-czapracki replied:
Done

szymon-czapracki commented on lib/smp_uwb/syscfg.yml:
```diff
@@ -1,12 +1,12 @@
 syscfg.defs:
-    NMGR_UWB_ENABLED:
+    SMP_UWB_ENABLED:
         description: 'Enable UWB transport for NewtMgr'
```
Done

ncasaril commented:
This only fails because the travis build assumes mynewt 1.7.
